portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -157,6 +157,74 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-139101. Verify missing references checkbox only prevents validation during Site import processes, not portlet import processes."
+	@priority = "3"
+	test CanImportJournalArticleWhenUncheckMissingReferences {
```

Comment:
It's better to change to **CanImportJournalArticleWhenUncheckMissingReferencesValidation**